### PR TITLE
Calendar Card

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_lovelace/calendar-card.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  required: true
+  description: calendar
+  type: string
+title:
+  required: false
+  description: The title of the card
+  type: string
+entities:
+  required: true
+  description: A list if calendar entities that will be displayed in the card
```
Comment (zsarnett) [inline source/_lovelace/calendar-card.markdown]:
```suggestion
  description: A list of calendar entities that will be displayed in the card
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @zsarnett 👍

**Review by frenck — APPROVED** [review]
Wrong button 😩

**Comment by zsarnett** [comment]
Thanks for the updates!

**Pull request merged**